examples/03_international_pricing/02_get_product_prices/query.graphql

```diff
@@ -1,11 +1,11 @@
-query allProducts @inContext(country: CA) {
+query allProducts($country: CountryCode) @inContext(country: $country) {
   products(first: 1) {
     edges {
       node {
         variants(first:1) {
           edges {
             node {
-              priceV2 {
+              price {
                 amount
                 currencyCode
```

Comment:
Could you add some additional fields to identify product and variant information. Maybe just titles